extension/github-notifications-preview.css

```diff
@@ -1,113 +1,114 @@
 :root:root .NPG-dropdown {
-    width: 750px;
-    padding: 0;
-    margin: 0;
-    color: rgb(36, 41, 46);
-    right: -10px;
-    z-index: 100;
+  width: 750px;
+  padding: 0;
+  margin: 0;
+  color: rgb(36, 41, 46);
+  right: -10px;
+  z-index: 100;
 }
+
 .NPG-dropdown.npg-has-groups {
-    border: none;
+  border: none;
 }
 
 /* This wrapping cannot be applied to the dropdown because it would clip the dropdown’s arrow */
 .NPG-dropdown > :first-child {
-    overflow: hidden;
-    border-radius: 4px; /* Matches GitHub’s */
+  overflow: hidden;
+  border-radius: 4px; /* Matches GitHub’s */
 }
 
 
 .NPG-dropdown .boxed-group {
-    margin-bottom: 20px;
+  margin-bottom: 20px;
 }
 
 .NPG-dropdown .boxed-group:last-child {
-    margin-bottom: 0 !important;
+  margin-bottom: 0 !important;
 }
 
 /* Removes text outline */
 .NPG-dropdown .boxed-group .text-green {
-    outline: none;
+  outline: none;
 }
 
 /* Loading animation */
 .NPG-loading .octicon-bell {
-    transform-origin: top;
-    animation: npg-ding 0.5s infinite ease-out;
+  transform-origin: top;
+  animation: npg-ding 0.5s infinite ease-out;
 }
 
 @keyframes npg-ding {
-    25% {
-        transform: rotate(10deg);
-        animation-timing-function: ease-in-out;
-    }
-    75% {
-        transform: rotate(-10deg);
-        animation-timing-function: ease-in;
-    }
+  25% {
+    transform: rotate(10deg);
+    animation-timing-function: ease-in-out;
+  }
+  75% {
+    transform: rotate(-10deg);
+    animation-timing-function: ease-in;
+  }
 }
 
 /* Keep icon clickable when modal is open */
 .NPG-container[open] + .notification-indicator {
-    z-index: 101;
+  z-index: 101;
 }
 
 /* Disable native tooltip when it has unread notifications */
 .Header .notification-indicator[data-ga-click$=":unread"]::after {
-    display: none;
+  /*display: none;*/
 }
```

Comment:
If code is not required, it should be dropped, not just commented out

Reply:
I was unsure why you put display none, so I left it out for your own consideration